packages/cfd/src/Containers/ctrader-derivx-trade-modal.tsx

```diff
@@ -81,19 +82,53 @@ const TradeModal = ({
     platform,
     is_mobile,
 }: TTradeModalProps) => {
-    const CTraderDescription = () => {
-        const platform_name = 'cTrader';
-        return (
-            <div className='cfd-trade-modal__login-specs-item'>
-                <Text className='cfd-trade-modal--paragraph'>
+    const {
+        ui,
+        client,
+        common,
+        traders_hub,
+        modules: { cfd },
+    } = useStore();
+
+    const { ctrader_accounts_list, trading_platform_available_accounts } = client;
+    const { setAccountType, toggleMT5TradeModal } = cfd;
+    const { setAppstorePlatform } = common;
+    const { openDerivRealAccountNeededModal } = ui;
+    const { selected_account_type, no_CR_account, is_real, has_any_real_account, getAccount } = traders_hub;
+
+    const total_balance = ctrader_accounts_list
+        ? ctrader_accounts_list.reduce((accumulator, ctrader_acc) => accumulator + (ctrader_acc?.balance ?? 0), 0)
+        : 0;
+
+    const message = {
+        header: (
+            <Text as='h2' weight='bold' className='cfd-trade-modal__expansion-panel--header'>
+                {localize('See important notes')}
+            </Text>
+        ),
+        content: (
+            <React.Fragment>
+                <Text size='xxs' line_height='xl' className='cfd-trade-modal__expansion-panel--content'>
+                    {localize('Use your Deriv account email and password to log in to cTrader.')}
+                </Text>
+                <div className='cfd-trade-modal__expansion-panel--divider' />
+                <Text size='xxs' line_height='xl' className='cfd-trade-modal__expansion-panel--content'>
                     <Localize
-                        i18n_default_text='Use your Deriv account email and password to login into the {{ platform }} platform.'
-                        values={{ platform: platform_name }}
+                        i18n_default_text='Manage up to {{max_count}} Deriv cTrader accounts (up to {{ strategy_count }} strategy accounts and 1 non-strategy account for payouts and commissions).'
+                        values={{
+                            max_count: trading_platform_available_accounts[0]?.max_count,
+                            strategy_count: Number(trading_platform_available_accounts[0]?.max_count) - 1,
```

Comment:
@hirad-deriv why you are reducing 1 from max_count?
What if a user already created for example 3 accounts?

Reply:
`max_count` does not change. `available_account` is the one that changes. You will always have 1 non-strategy account and `max_count - 1` strategy accounts.